.github/workflows/test-on-crc.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
       uses: actions/checkout@v4
       with:
         ref: ${{ github.sha }}
-    - uses: imjasonh/setup-ko@v0.7
+    - uses: imjasonh/setup-ko@v0.8
```

Comment:
@PuneetPunamiya, should this be using `ko-build/setup-ko@v0.8` like in `.github/workflows/reusable-e2e.yaml`?